src/main/webapp/app/components/tabs/CurationHistoryTab.tsx

```diff
@@ -50,6 +51,32 @@ const CurationHistoryTab = observer(({ historyData, getUsers, users, historyTabS
     };
   }, []);
 
+  const alterations = useMemo(() => {
+    const alts = new Set<string>();
+    for (const mutation of mutations || []) {
+      for (const alteration of mutation.alterations || []) {
+        alts.add(alteration.alteration);
+      }
+      mutation.name.split(',').forEach(alt => alts.add(alt.trim()));
+    }
+    return alts;
+  }, [mutations]);
+
+  const mutationOptions = useMemo(() => {
```

Comment:
Should this be sorted? Do we use any default sorting for Author Select?

Reply:
There is not default sorting for the author select. However, I have changed the author dropdown to be sorted alphabetically.

I decided to sort the mutations using the default sort we came up with. However, we can use any of our mutation sorts for this order, so let me know if you prefer another sort.

Reply:
The default sorting is a bit odd without the context within the collapsible. But we can leave it as it until someone complains.